src/core/types/decimal192.rs

```diff
@@ -1230,7 +1230,7 @@ mod test_decimal {
         test(f32::MIN as f64, "-340282346638528860000000000000000000000");
         test(123456789.87654321, "123456789.87654321");
         test(4.012_345_678_901_235, "4.012345678901235"); // precision lost
-        test(4.012_345_678_901_234_567_890, "4.012345678901235"); // Over 18 decimals is OK (precision lost)
+        test(4.012_345_678_901_235, "4.012345678901235"); // Over 18 decimals is OK (precision lost)
```

Comment:
this commit `bc1a546357f2e10bb05e0d3258dc677b826bbb29` looks like a `clippy fix` commit, is that correct? Then it might have been unintentional to change `4.012_345_678_901_234_567_890 ` -> `4.012_345_678_901_235`? But maybe it is correct to do the change, because clippy knows that `4.012_345_678_901_234_567_890` does not even fit in an `f64`?

Reply:
need to revert it a then the test is not correct